### ISSUE TITLE
Replace some HTML tags with semantic tags

### ISSUE THREAD

**Issue by anilsenay** [anilsenay/next-e-commerce#5]
For a better a11y(Accessibility) some div tags must be replace with semantic tags.

**Comment by dudzpedra** [comment]
Hello! still need help with that? I'm studying the Nextjs Commerce template (by Vercel) and would love to contribute!

**Comment by anilsenay** [comment]
@dudzpedra Of course, thank you! I couldn't move the project forward for a while. So community support could be very helpful.

**Comment by dudzpedra** [comment]
Created a PR for some changes in index.js, hope you like it. Just getting started here =)

**Comment by dudzpedra** [comment]
> Created a PR for some changes in index.js, hope you like it. Just getting started here =)

Deleted that PR because of the firebase information that shouldnt be out there.

I was thinking about maybe changing those tags to section tags

One section for the div.header and another one for the div.header buttons

What do you think?
